en/docs/get-started/how-to-guides/ai-data-mapping.md

```diff
@@ -0,0 +1,487 @@
+# Data Mapping using AI
+
+Did you know that the WSO2 Micro Integrator component of WSO2 Integrator comes with built-in data mapping capabilities to streamline your integration workflows?
+This data mapping solution brings you a powerful graphical interface, combined with AI-driven assistance, to perform all sorts of data mappings in mere seconds.
+
+In this guide, we are generating an API that processes weather data retrieved from the OpenWeather API for a mobile app. The app only requires a selected set of data from the API response and needs to modify some of the retrieved data to meet its requirements.
+
+## Let's get started!
+
+### Prerequisites
+
+You need Visual Studio Code (VS Code) with the <a target="_blank" href="https://marketplace.visualstudio.com/items?itemName=WSO2.micro-integrator">Micro Integrator for VS Code</a> extension installed.
+
+!!! Info
+    See the [Install Micro Integrator for VS Code]({{base_path}}/develop/mi-for-vscode/install-wso2-mi-for-vscode) documentation to learn how to install Micro Integrator for VS Code.
+
+### Step 1: Create an integration project
```

Comment:
Suggestion: IMO since these steps are already covered in guide "Generate integrations using AI", we don't have to take away main focus by giving manual steps. We can redirect user to that previous guide and highlight the AI datamapper function in this. I don't think they both covers same scenario having adependency between guides is not an issue. WDYT?

Reply:
AFAIK, these docs will be linked to the youtube videos. Therefore the main audience will be new users, hence it's better to give a complete guidance rather than referring to a different guide to cover some steps.

Reply:
The purpose of these "How to" guides is to bring to the reader's attention any new features recently added to the product and to give him a chance to give each feature a quick go. The said user can be an old user or someone who just came to check out the exact feature. Therefore, all the steps need to be outlined for each guide.